app/clients/freshdesk.py

```diff
@@ -1,12 +1,84 @@
 import json
-
 import requests
+
 from requests.auth import HTTPBasicAuth
+from typing import Dict, List, Union
 
 from flask import current_app
 
+from app.user.contact_request import ContactRequest
+
+
+__all__ = [
+    'Freshdesk'
+]
+
 
 class Freshdesk(object):
+
+    def _generate_description(self):
+        message = ''
```

Comment:
Could start with `message = self.contact.message` and remove the `else` statement if you feel like it